docs/configuration.md

```diff
@@ -2551,21 +2558,24 @@ Spark subsystems.
     <a href="streaming-kafka-0-10-integration.html">Kafka Integration guide</a>
     for more details.
   </td>
+  <td>1.3.0</td>
 </tr>
 <tr>
-    <td><code>spark.streaming.kafka.minRatePerPartition</code></td>
-    <td>1</td>
-    <td>
-      Minimum rate (number of records per second) at which data will be read from each Kafka
-      partition when using the new Kafka direct stream API.
-    </td>
+  <td><code>spark.streaming.kafka.minRatePerPartition</code></td>
```

Comment:
Same here.

Reply:
Thanks. I will document them in a separate PR.

Reply:
And here also...

Reply:
I revert it temporarily too.

Reply:
@beliefer, I think you had to clarify you're working on this for each file as the logical group. Seems like that missing context is causing the miscommunication here with @gaborgsomogyi.